components/sun_gtil2.rst

```diff
@@ -0,0 +1,123 @@
+SUN_GTIL2 Grid Tie Inverter
+===========================
+
+.. seo::
+    :description: Instructions for reading data from a SUN-1000G2 grid tie inverter using ESPHome
+    :keywords: sun-1000g2, sun-2000g2, sun1000, sun2000, sun-1000gtil2, sun-2000gtil2
+
+The ``sun_gtil2`` component allows you to get voltage, power and temperature readings from your ``SUN-1000G2``
+or ``SUN-2000G2`` grid tie inverter. This is done by reading the UART data transmitted from the inverter's
+controller board to the display board.
+
+.. note::
+
+    You need to open your inverter to apply this modification. Use at your own risk! If you want to connect your
+    inverter via the (more limited) external RS232 interface you should use the
+    :doc:`Modbus </components/modbus_controller>` component instead.
+
+.. figure:: images/sun_gtil2_controller_board.png
+    :align: center
+    :width: 50.0%
+
+    Pinout of the inverter's controller board
+
+.. figure:: images/sun_gtil2_display_board.png
+    :align: center
+    :width: 50.0%
+
+    Pinout of the inverter's display board
+
+.. figure:: images/sun_gtil2_schematic.png
+    :align: center
+    :width: 75.0%
+
+    Simplified connection diagram
+
+As the data is read from the inverter using UART, you need to have an :ref:`UART bus <uart>` in your
+configuration with the ``rx_pin`` connected to the TX pin of the inverter's controller board. Additionally, you
+need to set the ``baud_rate`` to 9600.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    uart:
+      - id: control_to_display
+        rx_pin: GPIO16
+        baud_rate: 9600
+
```

Comment:
Dependent components are not to be included in examples
```suggestion
```

Reply:
Should I put that somewhere else or just leave it out completely?

Reply:
Just leave it out, its up to the user to read the docs on uart and configure it according to their physical board and pins.